src/Mod/Path/PathScripts/PathMillFace.py

```diff
@@ -257,12 +278,12 @@ def isOverlap(fMn, fMx, eMn, eMx):
             fYMax = face.BoundBox.YMax
             # fZMin = face.BoundBox.ZMin
             fZMax = face.BoundBox.ZMax
-            
+
             if fZMax > eZMin:
-                if isOverlap(fXMin, fXMax, eXMin, eXMax) is True:
-                    if isOverlap(fYMin, fYMax, eYMin, eYMax) is True:
+                if isOverlap(fXMin, fXMax, eXMin, eXMax):
+                    if isOverlap(fYMin, fYMax, eYMin, eYMax):
```

Comment:
This is a triple `and`, so maybe instead of indenting three times, you can use something like this.
```py
if (fZMax > eZMin
        and isOverlap(fXMin, fXMax, eXMin, eXMax)
        and isOverlap(fYMin, fYMax, eYMin, eYMax)):
    included.append(face)

return included
```

Setting `incl` is superfluous because it is just a boolean. The last `if` is the fourth expression in the `and` chain. You can just remove it.